pepito_alonajane/countries.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="countries.css">
+    <title>Countries</title>
+</head>
+<body>
+    <div class="container">
+        <h1>Countries of the World</h1>
+        <input type="text" id="country_Input" 
+                placeholder="Please enter a country">
+        <button onclick="searchCountry()">Search</button>
+        <div class="country-details" id="country_Details"></div>
+        <div class="same-region-countries" id="same_Region_Countries"></div>
```

Comment:
-IDs should be written in snake case. All lower case separated by underscore.
-do not put id and class in one div
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.p